components/navbar.vue

```diff
@@ -54,8 +66,25 @@ export default defineComponent({
   methods: {
     logout() {
       this.$store.commit('setToken', null)
-      this.$router.replace('/login')
+      this.$router.replace('/login/')
     },
+    async getDetails() {
+      let t;
+      let token = this.$store.getters.token;
+      await this.$axios({
+        headers: {
+            Authorization: 'Token ' + token,
+        },
+        url: 'https://todo-app-csoc.herokuapp.com/' + 'auth/profile/',
+        method: 'get',
+      }).then(function({data, status}) {
+      t = JSON.parse(JSON.stringify(data))
+      //console.log(t)
```

Comment:
Again copying some mistake from someone else its not a framework related shouldn't have done that